### PR TITLE
Feature/edge 194

### PR DESCRIPTION
**Pull request by ionyshch-prismsl**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -123,7 +123,18 @@ function(buildSdk)
         set (CONNECT_BUILD_TYPE ${CMAKE_BUILD_TYPE})
     endif ()
 
-    set(SDK_VER "${ConnectSDK_VERSION_MAJOR}.${ConnectSDK_VERSION_MINOR}.${ConnectSDK_VERSION_REVISION}")
+    if (DEFINED ENV{BUILD_NUMBER})
+        set (CONNECT_BUILD_NUMBER $ENV{BUILD_NUMBER})
+    else ()
+        message (WARNING "BUILD_NUMBER env variable not set, using value 0 for build number")
+        set (CONNECT_BUILD_NUMBER 0)
```
Comment (dbus-prsm) [inline CMakeLists.txt:130]: Can we use string (say "custom") for such builds here?

File: build.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #	-t|--type=Debug|Release specify build type
 # 	--delivery              create delivery package
 
-# uncomment for debugging
+# uncomment for debugging or call with "bash -uvx build.sh <build-params>"
```
Comment (dbus-prsm) [inline build.sh]: I think you should indicate that this is for debugging the script, not SDK.

File: CMakeLists.txt (hunk 1 above)
Comment (ionyshch-prismsl) [reply CMakeLists.txt:130]: We can. But 'custom' isn't a number and will look weird 01.00.001.custom, so I chose to use invalid build number resulting in 01.00.001.00000

File: build.sh (hunk 2 above)
Comment (ionyshch-prismsl) [reply build.sh]: Ok

**Review by bvnp44 — APPROVED** [review]

**Pull request merged**